### DISCUSSION TITLE
unable to build my custom plugin as a .jar

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15362] [Q&A] [answered]
i was able to locate the "example.class".
I would like the gradle to generate a .jar for me to integrate into my runelite client without having to launch the IDE every time i want to use my custom plugin.

this is not a plugin i want to publicly release, it is just for myself and friends

**Comment by Nightfirecat** [comment, marked as answer]
Running `mvn package` generates a shaded jar that you can run directly.
